File: 백준/Bronze/10872. 팩토리얼/팩토리얼.py
N = int(input())
fac = 1

# 0이 아닌 N에 대한 N1 처리
for n in range(N, 0, -1): # 이렇게 범위설정으로 0입력되면 for문 돌지않고 1로 초기화한 fac 변수값 출력
    fac *= n
print(fac)

File: 백준/Bronze/10872. 팩토리얼/팩토리얼.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int fac = 1;
        for (int n = N; n > 0 ; n--) {
            fac *= n;
        }
        System.out.println(fac);
    }
}